from_mwt_ds/DataScience/vw_executor/vw.py

```diff
@@ -125,10 +140,87 @@ def __init__(self,
         self.status = ExecutionStatus.NotStarted
         self.model_file = model_file
         self.model_folder = model_folder
+        self._order_position = order_position
         self._no_run = no_run
         self.args = self._prepare_args(self.job.cache)
         self.start_time = None
         self.end_time = None
+    
+    def create_human_readeable_symlink(self, translate_output: Dict[str, str] = {"-p": "predictions.txt", "-f": "final_regressor.vwmodel", "--extra_metrics": "extra_metrics.json"}, current_time: Optional[str] = None) -> None:
```

Comment:
it is weird to have current_time of string type - maybe we can call it tag or something like that?